Add to documentation that HTML should not be used in instrument fields (#6267)
aces/Loris: modules/dataquery/README.md

###### Import the LORIS instrument data

> This step is optional and not required if only the MRI portion of LORIS is used: \

Reviewer: I would prefer to indicate when it is required. Other options might be invalidated in the future.
Similar change should also be applied to the MRI section below.
```suggestion
This step is optional and is required only if instruments are used: \
```

Reviewer 2: I believe the suggestion is not the original meaning, and contradictory.  @PapillonMcGill

Reviewer 3: @lingma you are right that it is not the original meaning, but @PapillonMcGill 's change is also more accurate and so better

Reviewer 3: however not relevant to this PR which I will close.
